Feature: reconcile search with designs [NP-895]

davidsauntson opened the pull request:
Reason for change:
Designs for search had changed since they were first implemented in the design system.  This PR updates the style of the search bar to match those in the design file.

Changes:
 - remove icon in mobile search button and replace with text
 - added VRT for expanded header
 - update VRT references

davidrapson commented on scss/6-components/_navigation.scss:
```diff
@@ -1,6 +1,5 @@
 @import 'navigation_greedy-nav';
 
-nav,
```
💯

davidrapson reviewed (dismissed):
👍

soniaturcotte reviewed (approved):
Looks good for search page! 👍  thanks for doing that. Just to confirm this won't affect the header?

davidsauntson commented:
> Looks good for search page! 👍 thanks for doing that. Just to confirm this won't affect the header?

@soniaturcotte Collapsed it is the same, expanded it will show 'Search' instead of the icon:
![image](https://user-images.githubusercontent.com/2331893/94458517-f5382700-01ad-11eb-9c86-623391c50eef.png)

davidsauntson commented:
> > Looks good for search page! 👍 thanks for doing that. Just to confirm this won't affect the header?
> 
> @soniaturcotte It will show Search instead of the icon in the header on desktop as it currently does.
> 
> On mobile, collapsed it is the same, expanded it will show 'Search' instead of the icon.
> ![image](https://user-images.githubusercontent.com/2331893/94458517-f5382700-01ad-11eb-9c86-623391c50eef.png)

I've changed this now so it's back to the search icon on mobile, and added a VRT to cover this.

davidrapson reviewed (approved):
👍 Looks great. 😍 Thank you for removing the default element styles
